corehq/form_processor/tests/test_form_dbaccessor.py

```diff
@@ -415,6 +431,19 @@ def test_soft_delete(self):
             form = FormAccessors(DOMAIN).get_form(form_id)
             form.delete()
 
+    def test_update_responses(self):
+        formxml = FormSubmissionBuilder(
+            form_id='123',
+            form_properties={'breakfast': 'toast', 'lunch': 'sandwich'}
+        ).as_xml_string()
+        xform = submit_form_locally(formxml, DOMAIN).xform
+
+        updates = {'breakfast': 'fruit'}
+        FormProcessorInterface(DOMAIN).update_responses(xform, updates, 'user1')
+        form = FormAccessors(DOMAIN).get_form(xform.form_id)
+        self.assertEqual('fruit', form.form_data['breakfast'])
+        self.assertEqual('sandwich', form.form_data['lunch'])
```

Comment:
If you're doing this same test on both backends you should just need to add it once. Looking at this file I think you should add it to `FormAccessorsTests` (which is here). This class get's extended by `FormAccessorsTestsSQL` with the `@use_sql_backend` decorator so it will run all the same test on the SQL backend.

Reply:
nice, updated